packages/gatsby/src/internal-plugins/webpack-theme-component-shadowing/gatsby-node.js

```diff
@@ -4,14 +4,14 @@ exports.onCreateWebpackConfig = (
   { store, stage, getConfig, rules, loaders, actions },
   pluginOptions
 ) => {
-  const { config, program } = store.getState()
+  const { program, themes } = store.getState()
 
-  if (config.__experimentalThemes) {
+  if (themes) {
```

Comment:
Themes will actually be defined here, even if it's not used right? I think we want:

```suggestion
   if (themes.themes) {
```